ArduinoCode/Pedal_Box/Pedal_Box.ino

```diff
@@ -134,6 +134,19 @@ void readPotentiometers() {
   accelPin1Val = analogRead(ACCEL_PIN1); 
   accelPin2Val = analogRead(ACCEL_PIN2);
 
+  if(accelPin1Val || accelPin2Val == 0){
```

Comment:
```suggestion
  if(accelPin1Val == 0 || accelPin2Val == 0){
```
Pretty sure that the left and right side of the double pipe or operator both need to be booleans.